src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
Hmm, good point; okay, let me think about it a bit more. Please ping me early next week if I forget to respond though :smile: